sycl/doc/extensions/C-CXX-StandardLibrary/DeviceLibExtensions.rst

```diff
@@ -32,3 +32,184 @@ Arguments:
+   double __devicelib_cabs(double __complex__ z);
+   float  __devicelib_cabsf(float __complex__ z);
+   double __complex__ __devicelib_cproj(double __complex__ z);
+   float  __complex__ __devicelib_cprojf(float __complex__ z);
+   double __complex__ __devicelib_cexp(double __complex__ z);
+   float  __complex__ __devicelib_cexpf(float __complex__ z);
+   double __complex__ __devicelib_clog(double __complex__ z);
+   float  __complex__ __devicelib_clogf(float __complex__ z);
+   double __complex__ __devicelib_cpow(double __complex__ x, double __complex__ y);
+   float  __complex__ __devicelib_cpowf(float __complex__ x, float __complex__ y);
+   double __complex__ __devicelib_cpolar(double x, double y);
+   float  __complex__ __devicelib_cpolarf(float x, float y);
+   double __complex__ __devicelib_csqrt(double __complex__ z);
+   float  __complex__ __devicelib_csqrtf(float __complex__ z);
+   double __complex__ __devicelib_csinh(double __complex__ z);
+   float  __complex__ __devicelib_csinhf(float __complex__ z);
+   double __complex__ __devicelib_ccosh(double __complex__ z);
+   float  __complex__ __devicelib_ccoshf(float __complex__ z);
+   double __complex__ __devicelib_ctanh(double __complex__ z);
+   float  __complex__ __devicelib_ctanhf(float __complex__ z);
+   double __complex__ __devicelib_csin(double __complex__ z);
+   float  __complex__ __devicelib_csinf(float __complex__ z);
+   double __complex__ __devicelib_ccos(double __complex__ z);
+   float  __complex__ __devicelib_ccosf(float __complex__ z);
+   double __complex__ __devicelib_ctan(double __complex__ z);
+   float  __complex__ __devicelib_ctanf(float __complex__ z);
+   double __complex__ __devicelib_cacos(double __complex__ z);
+   float  __complex__ __devicelib_cacosf(float __complex__ z);
+   double __complex__ __devicelib_casinh(double __complex__ z);
+   float  __complex__ __devicelib_casinhf(float __complex__ z);
+   double __complex__ __devicelib_casin(double __complex__ z);
+   float  __complex__ __devicelib_casinf(float __complex__ z);
+   double __complex__ __devicelib_cacosh(double __complex__ z);
+   float  __complex__ __devicelib_cacoshf(float __complex__ z);
+   double __complex__ __devicelib_catanh(double __complex__ z);
+   float  __complex__ __devicelib_catanhf(float __complex__ z);
+   double __complex__ __devicelib_catan(double __complex__ z);
+   float  __complex__ __devicelib_catanf(float __complex__ z);
+   double __complex__ __devicelib_muldc3(double a, double b, double c, double d);
+   float  __complex__ __devicelib_mulsc3(float a, float b, float c, float d);
+   double __complex__ __devicelib_divdc3(double a, double b, double c, double d);
+   float  __complex__ __devicelib_divsc3(float a, float b, float c, float d);
+
+Semantic:
+Those __devicelib_* functions have the same semantic as corresponding complex math functions
+from <complex.h>. Please refer to: https://en.cppreference.com/w/c/numeric/complex for details.
```

Comment:
I think it's better to refer to the C++/C standard rather than cppreference web-site.

Reply:
Yes, I will update them all.

Reply:
Hi, Alexey.
I changed to "Please refer to ISO/IEC 14882:2011" as C++ standard document is not free, we can provide a link to online doc here.